src/lib.rs

```diff
@@ -141,9 +137,6 @@ pub struct TrayIconAttributes {
     ///     Setting an empty [`Menu`](crate::menu::Menu) is enough.
     pub icon: Option<Icon>,
 
-    /// Tray icon temp dir path. **Linux only**.
```

Comment:
There is a few more documentation for linux only, like `tooltip` unsupported, let's update these as well